sklearn/mixture/gmm.py

```diff
@@ -112,7 +114,7 @@ def sample_gaussian(mean, covar, covariance_type='diag', n_samples=1,
     return (rand.T + mean).T
 
 
-class GMM(BaseEstimator):
+class _GMMBase(_MixtureBase):
```

Comment:
yes but just the code for GaussianMixture (not the variational Bayes sub classes).